integrations/hello-world-ng6/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "hello-world-ng6",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve --hmr",
+    "test": "ng build",
+    "pretest": "yarn --pure-lockfile",
+    "postinstall": "cpx -v -C \"../../@ngxs/**/*\" \"node_modules/@ngxs/\""
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "6.1.1",
+    "@angular/common": "6.1.1",
+    "@angular/compiler": "6.1.1",
+    "@angular/core": "6.1.1",
+    "@angular/forms": "6.1.1",
+    "@angular/http": "6.1.1",
+    "@angular/platform-browser": "6.1.1",
+    "@angular/platform-browser-dynamic": "6.1.1",
+    "@angular/router": "6.1.1",
+    "core-js": "^2.5.5",
+    "karma-jasmine": "^1.1.1",
```

Comment:
Could you remove all unneeded packages (like `karma*`, `ts-node`, `codelyzer` etc...) and only those that are necessary for this build? Also, group all packages inside the `devDependencies` section (like in the root `package.json`)